include/zephyr/bluetooth/audio/audio.h

```diff
@@ -824,8 +824,8 @@ enum bt_audio_dir {
  *
  * @param freq The assigned numbers frequency to convert.
  *
- * @retval -EINVAL if arguments are invalid.
- * @retval The converted frequency value in Hz.
+ * @retval -EINVAL Arguments are invalid.
+ * @retval frequency The converted frequency value in Hz.
```

Comment:
You want this one listed first I think, i.e. the most likely return value first (applies throughout the PR)

Related, and since we don't have guidelines just yet and I very much value your thought on this topic and the fact we can use this PR as a way to settle on best practices for documenting return values, how about the "non-distinctive" value be described using `@return`? Basically, this: https://docs.rtems.org/branches/master/eng/coding-doxygen.html#function-declarations

Reply:
> how about the "non-distinctive" value be described using @return

So this is suggesting to use `@retval` for specific values, and `@return` for values that may range?

So in this example this would be something like 

```c
 * @retval -EINVAL Arguments are invalid.
 * 
 * @return The converted frequency value in Hz.
```

I'm not opposed to it, depending on how it would render in the built docs :) It does make sense to only use `@retval` for specific values. Speaking of, should we use `- @ref EINVAL`? or add something like a `@errno{EINVAL}`? 


I think it makes sense to propose an RFC to discuss this

Reply:
> Speaking of, should we use `- @ref EINVAL`? or add something like a `@errno{EINVAL}`?

Good question. Given that retval accepts ONE token representing the return value, the former likely won't work, and not sure about the latter either. That being said, I like the idea as making it easier to navigate to the list of available errnos can only be goodness.

+1 to address the next iteration (combining usage of `@return` and `@retval`) in a follow-up RFC